macos/FluentUI/Button/Button.swift

```diff
@@ -228,16 +233,15 @@ open class Button: NSButton {
 		}
 	}
 
-	private var restBackgroundColor: NSColor {
-		return fillColor
-	}
-
+	@objc public var restBackgroundColor: NSColor? = nil
+	@objc public var cornerRadius: CGFloat = 3
+	
 	private var hoverBackgroundColor: NSColor {
-		return fillColor.withSystemEffect(.rollover)
+		return restBackgroundColor?.withSystemEffect(.rollover) ?? fillColor.withSystemEffect(.rollover)
 	}
 
 	private var pressedBackgroundColor: NSColor {
-		return fillColor.withSystemEffect(.pressed)
+		return restBackgroundColor?.withSystemEffect(.pressed) ?? fillColor.withSystemEffect(.pressed)
```

Comment:
I think you need to sync to the latest master, as these shouldn't be included as part of your change

Reply:
These were intended changes.